config/slashCommandPermissions.js

```diff
@@ -0,0 +1,35 @@
+const {ID_CODE_COUNSELOR, ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN} = process.env;
+
+// Commands with .setDefaultPermission(false) need to be enabled by adding role IDs to array.
+
+const commandRoles = new Map([
+  ['ping', [ID_CODE_COUNSELOR, ID_ADMIN]],
+  ['stats', [ID_MODERATOR, ID_ADMIN, ID_SUPER_ADMIN]],
+]);
```

Comment:
I'm on the fence about adding these slash commands in like this. On the one hand, these are not supposed to have any role restrictions and are duplicate code from the community bot, so I don't like the idea of actually committing them in this repo.

On the other hand, they prove that slash commands work side by side without having to wait for the other slash commands to be built out... 🤔

Reply:
The permissions were setup like this only for easy testing. If these 3 commands are to be deployed then it makes most sense to change the permissions to how they are under the text based permission system cc!ping is for code counselor and above. cc!stats/cc!helpcenter have no restrictions. But perhaps until there is a clear plan for deployment/hosting of the community bot it may be a good idea to include them?

Reply:
I changed the role permissions for these 3 commands to match the cc! text commands. The only command with a restriction is now /ping.